exporter.go

```diff
@@ -21,45 +22,42 @@ type Exporter struct {
 	UID, GID     int
 }
 
-func (e *Exporter) Export(layersDir, appDir string, runImage, origImage image.Image, launcher string, stack StackMetadata) error {
+func (e *Exporter) Export(layersDir, appDir string, runImage, origImage image.Image, launcher string, stack metadata.StackMetadata) error {
 	var err error
 
-	metadata := AppImageMetadata{}
+	meta := metadata.AppImageMetadata{}
```

Comment:
In case it's helpful, I've seen `md` used as a common short name for metadata elsewhere.